be/src/olap/rowset/segment_v2/segment_iterator.cpp

```diff
@@ -1109,8 +1161,9 @@ Status SegmentIterator::_init_inverted_index_iterators() {
         int32_t unique_id = _opts.tablet_schema->column(cid).unique_id();
         if (_inverted_index_iterators.count(unique_id) < 1) {
             RETURN_IF_ERROR(_segment->new_inverted_index_iterator(
-                    _opts.tablet_schema->column(cid), _opts.tablet_schema->get_inverted_index(cid),
-                    _opts, &_inverted_index_iterators[unique_id]));
```

Comment:
so here is a bug fixed here?

Reply:
yes, it's an old bug unrelated to this pr.